x-pack/plugins/observability/public/components/shared/exploratory_view/components/series_color_picker.tsx

```diff
@@ -29,9 +35,9 @@ export function SeriesColorPicker({ seriesId, series }: { seriesId: number; seri
 
   const button = (
     <EuiToolTip content={EDIT_SERIES_COLOR_LABEL}>
-      <ToolbarButton size="s" onClick={() => setIsOpen((prevState) => !prevState)} hasArrow={false}>
-        <EuiIcon type="dot" size="l" color={color} />
-      </ToolbarButton>
+      <EuiButtonEmpty size="s" onClick={() => setIsOpen((prevState) => !prevState)} flush="both">
+        <EuiIcon type="stopFilled" size="l" color={color} />
+      </EuiButtonEmpty>
```

Comment:
I think we should continue using ToolbarButton here, it provides better user interaction experience. 

Though it's @liciavale call to make. 

![image](https://user-images.githubusercontent.com/3505601/133580230-0c579f96-f974-4f11-a17f-8f23f6df5183.png)

Reply:
I removed it for the sole purpose of removing the border, which is not present in the design.